tools/cmd/chronogen/pkg_metadata.go

```diff
@@ -53,6 +53,7 @@ func (p pkgMetadata) Swagger() string {
 	return inflect.Camelize(fmt.Sprintf("%s_%s", p.Type, p.APIVersion))
 }
 
+// Model returns the model with the api version uncapitolized, e.g. "Configv1".
```

Comment:
nit: typo, uncapitalized

Reply:
jesus christ lol how did i mess that one up